contrib/cirrus/runner.sh

```diff
@@ -61,6 +61,9 @@ _run_setup() {
     # VM's come with the distro. skopeo package pre-installed
     dnf erase -y skopeo
 
+    # Required for testing the SIF transport
+    dnf install -y fakeroot squashfs-tools
```

Comment:
@cevich, could we add these two packages to the CI images?

Reply:
I can but...the images are very bloated as-is, and these are tools only needed for one test in one repo. I think in this case it might be worth a compromise to just live with some runtime installs...as much as it pains me.  (The bloat really is bad).